pkg/crd/markers/validation.go

```diff
@@ -222,6 +224,6 @@ func (m Type) ApplyToSchema(schema *v1beta1.JSONSchemaProps) error {
 func (m Type) ApplyFirst() {}
 
 func (m Nullable) ApplyToSchema(schema *v1beta1.JSONSchemaProps) error {
-	schema.Nullable = bool(m)
+	schema.Nullable = true
```

Comment:
why true? Can we write `nullable: false`?

Reply:
I think currently `// +nullable` is only used to set `nullable: true`, and as we've seen a lack of this field already implies that it cannot be nullable. Also, if we do want to allow `// +nullable=false` then all other existing usages of the nullable tag would have to be updated to `// +nullable=true`, or they would raise a missing parameter error when being parsed.

Reply:
ok, fine with me.

Reply:
you could technically make this a bool pointer if you wanted that